src/docfx/lib/moniker/MonikerConstants.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Docs.Build
+{
+    internal class MonikerConstants
+    {
+        internal const string NoneVersion = "NONE_VERSION";
```

Comment:
avoid `Constants` that contains magic strings. I believe this is the only magic string for monikers, so put it inline.